tests/dags/providers/provider_api_scripts/test_smithsonian.py

```diff
@@ -641,139 +441,17 @@ def test_extract_meta_data_dnr_fields(input_ft, input_dnr, expect_meta_data):
 )
 def test_extract_tags(input_is, expect_tags):
     input_row = {"test": "row"}
-    get_is = patch.object(si, "_get_indexed_structured_dict", return_value=input_is)
+    get_is = patch.object(
+        ingester, "_get_indexed_structured_dict", return_value=input_is
+    )
     with get_is as mock_is:
-        actual_tags = si._extract_tags(input_row)
+        actual_tags = ingester._extract_tags(input_row)
     mock_is.assert_called_once_with(input_row)
     assert actual_tags == expect_tags
 
 
 @pytest.mark.parametrize(
-    "input_row,expect_dnr_dict",
-    [
-        ({}, {}),
-        ({"content": {"key": {"key2", "val2"}}}, {}),
-        ({"noncontent": {"descriptiveNonRepeating": {"key2": "val2"}}}, {}),
-        ({"content": {"descriptiveNonRepeating": {"key2": "val2"}}}, {"key2": "val2"}),
-    ],
-)
-def test_get_descriptive_non_repeating_dict(input_row, expect_dnr_dict):
-    actual_dnr_dict = si._get_descriptive_non_repeating_dict(input_row)
-    assert actual_dnr_dict == expect_dnr_dict
-
-
-@pytest.mark.parametrize(
-    "input_row,expect_ind_struc_dict",
-    [
-        ({}, {}),
-        ({"content": {"key": {"key2", "val2"}}}, {}),
-        ({"noncontent": {"indexedStructured": {"key2": "val2"}}}, {}),
-        ({"content": {"indexedStructured": {"key2": "val2"}}}, {"key2": "val2"}),
-    ],
-)
-def test_get_indexed_structured_dict(input_row, expect_ind_struc_dict):
-    actual_ind_struc_dict = si._get_indexed_structured_dict(input_row)
-    assert actual_ind_struc_dict == expect_ind_struc_dict
-
-
-@pytest.mark.parametrize(
-    "input_row,expect_freetext_dict",
-    [
-        ({}, {}),
-        ({"content": {"key": {"key2", "val2"}}}, {}),
-        ({"noncontent": {"freetext": {"key2": "val2"}}}, {}),
-        ({"content": {"freetext": {"key2": "val2"}}}, {"key2": "val2"}),
-    ],
-)
-def test_get_freetext_dict(input_row, expect_freetext_dict):
-    actual_freetext_dict = si._get_freetext_dict(input_row)
-    assert actual_freetext_dict == expect_freetext_dict
-
-
-@pytest.mark.parametrize(
-    "required_type,default_input",
-    [
-        (str, ""),
-        (int, 0),
-        (float, 0.0),
-        (complex, 0j),
-        (list, []),
-        (tuple, ()),
-        (dict, {}),
-        (set, set()),
-        (bool, False),
-        (bytes, b""),
-    ],
-)
-def test_check_type_with_defaults(required_type, default_input):
-    assert si._check_type(default_input, required_type) == default_input
-
-
-@pytest.mark.parametrize(
-    "required_type,good_input",
-    [
-        (str, "abc"),
-        (int, 5),
-        (float, 1.2),
-        (complex, 3 + 2j),
-        (list, ["a", 2, 0, False]),
-        (tuple, (1, 0, False)),
-        (dict, {"key": "val", "f": False}),
-        (set, {1, False, "abc"}),
-        (bool, True),
-        (bytes, b"abc"),
-    ],
-)
-def test_check_type_with_truthy_good_inputs(required_type, good_input):
```

Comment:
Should we leave the `check_type` tests in for now?

Reply:
Copied the tests for some other internal methods as well. I thought I did before!